README.md

```diff
@@ -80,3 +82,54 @@ $ npm install
 $ npm run clean
 $ npm run build
 ```
+
+## For server-side rendering via node.js
+
+Copy `xypic.js` to your npm project, and create a `mathjax.js` in the same directory containing:
+
+```js
+import { readFileSync, writeFileSync } from 'fs';
+import { argv } from 'yargs';
+
+import * as mathjax from 'mathjax-full';
+
+mathjax
+  .init({
+    options: {
+      typesetError: (_, math, err) => console.log(math.math + ': ' + err),
+    },
+    loader: {
+      paths: { mathjax: 'mathjax-full/es5', custom: '.' },
+      require: require,
+      load: ['input/tex-full', 'output/chtml', '[custom]/xypic'],
+    },
+    tex: {
+      packages: { '[+]': ['xypic'] },
+      inlineMath: [['$', '$']],
```

Comment:
You don't need this for your example, as this is a user preference.  your use case may not be everyone's.  If you do keep it, then probably include `['\\(', '\\)']` as well, so the default delimiters work,